latex/content/tlmgr.tex

```diff
@@ -0,0 +1,36 @@
+\section{\texttt{tlmgr}}
+
+\begin{frame}[fragile]{
+  \texttt{tlmgr}
+}
+  \begin{itemize}
+    \item \TeX-Live kommt mit einem eigenen Verwaltungstool
+    \item Neue Pakete installieren, updaten, suchen, ...
+  \end{itemize}
+
+  \begin{block}{Ein neues Paket installieren}
+    Wenn man die Dokumentation auch haben möchte, \texttt{--with-doc} nutzen:
+    \begin{center}
+      \texttt{tlmgr install [--with-doc] <name>}
```

Comment:
Ist es möglich die gewünschten packages in einer txt, yaml oder was auch immer abzuspeichern (im coolsten Fall sogar mit einer Versionsnummer) und diese dann zu installieren anstatt jedes package, das gebraucht wird manuell zu installieren?

Reply:
Habe die Docs aber gerade überflogen. Da sah es nicht so aus.

Reply:
Das würde ja heißen das tex sowas wie dependencies Management hätte...

Reply:
Was für die Reproduzierbarkeit alter Arbeiten echt cool wäre, gibt es aber nicht oder?

Reply:
Das ist kein Problem, weil du alle alten TexLive Versionen installieren kannst. 

Anfang des Jahres wird immer das komplette TeXLive des Vorjahres "eingefroren" und diese eingefroren Versionen kannst du installieren.

Das läuft aber halt auf der TeXLive Ebene und einmal im Jahr. Nicht mit einzelnen Paket Versionen

Reply:
https://tex.stackexchange.com/a/25092/59716

Reply:
Okay, danke.